libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtils.kt

```diff
@@ -481,6 +483,48 @@ internal object MapboxRouteLineUtils {
         }
     }
 
+    /**
+     * Returns the color that is used to represent traffic congestion on inactive legs of the primary route.
+     *
+     * @param congestionValue as string value coming from the DirectionsRoute
+     * be returned or the color for an alternative route.
```

Comment:
I can't parse these 2 lines, do they belong together?

Reply:
Refactor/copy error. This doesn't help much anyway, I'll just remove it.